source/to_cpp1.h

```diff
@@ -6719,7 +6719,9 @@ class cppfront
 
             //  Emit "auto" for deduced types (of course)
             if (type->is_wildcard()) {
-                assert(n.initializer);
+                if(!n.initializer) {
+                    return;
```

Comment:
Should there be an error recorded here, like at line 6735?

Reply:
there would be a double reporting of the error since sema already records it.

Reply:
good eye though